### ISSUE TITLE
The number of pretraining epochs on PCQM4Mv2

### ISSUE THREAD

**Issue by Data-reindeer** [shehzaidi/pre-training-via-denoising#1]
Hi Sheheryar,
Thanks for your nice work ! I have a question about the number of pre-training epochs. From the Appendix in the paper, the gradient steps for pretraining is 3e5, while form [ET-PCQM4MV2.yaml](https://github.com/shehzaidi/pre-training-via-denoising/blob/main/examples/ET-PCQM4MV2.yaml) it is set to 70 epochs (~2e6 gradient steps). 
Does that mean the pretraining is stopped by early stopping strategy? Have you tried what happens if you train for more epochs?

**Comment by shehzaidi** [comment]
Hi @Data-reindeer, thanks for the message! In the config file, `num_steps: 400000` by default ([see here](https://github.com/shehzaidi/pre-training-via-denoising/blob/3e10c355f1473097a581e150093a8c74dbd261b2/examples/ET-PCQM4MV2.yaml#L40)), which is when training should end. PyTorch Lightning will end training when the maximum number of steps is reached ([see here](https://pytorch-lightning.readthedocs.io/en/stable/common/trainer.html#max-steps)), which in this case is before we reach the 30 epochs as specified by `num_epochs: 30`. We don't use any early stopping during pre-training and typically used 200-400k gradient steps depending on the architecture, which tends to be enough for plateauing loss curves.
